index.js

```diff
@@ -16,43 +52,57 @@ module.exports = {
       name: options.name,
 
       defaultConfig: {
-        configuredServices: {
-          bugsnag: {
-            url: 'http://notify.bugsnag.com/deploy'
+        configuredServices: function(context) {
+          return {
+            bugsnag: {
+              url: 'http://notify.bugsnag.com/deploy',
+              method: 'POST',
+              headers: {},
+              body: function() {
+                var apiKey = this.apiKey;
+
+                if (!apiKey) { return; }
```

Comment:
That's the way it's working right now without the warning. Bugsnag is kind of special as it preconfigures  to notify on `didActivate`. We can remove that of course and don't default to `didActivate`.